src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
WTH is this?

Reply:
It was created during `npm run start`. I'll look into tsconfig.json, how it's handled in other packages.